public/app/features/dashboard/components/Inspector/InspectDataTab.tsx

```diff
@@ -73,19 +94,43 @@ export class InspectDataTab extends PureComponent<Props, State> {
     return data;
   }
 
+  applyTimeFormatting = (data: DataFrame[]): DataFrame[] => {
+    for (const frame of data) {
+      for (const field of frame.fields) {
+        if (field.type === FieldType.time) {
+          field.display = getDisplayProcessor({
+            field,
+            theme: config.theme,
+          });
+        }
+      }
+    }
+
+    return data;
+  };
```

Comment:
Think I understand the reasons, but this feels wrong, this is handled in applyFieldOverrides, should be able to use that somehow. And if no field config options are applied we only show raw data.

Reply:
yeah, i'm about to change this, as I hate ^^ approach

Reply:
@torkelo this is now updated and ready for another take from you :)